guides/common/modules/con_content-management-types-overview.adoc

```diff
@@ -1,7 +1,12 @@
 [id="Content_Management_Types_Overview_{context}"]
 = Content Management Types Overview
 
+ifdef::satellite[]
 With {ProjectName}, you can manage the following Red Hat content types:
```

Comment:
I'll let others weigh in, but IMHO it can always manage non-Red Hat content types. Line 47 is wrong and it can manage "files" Pulp 2 called it the ISO store, but Pulp 3 just names it files because that's what it is. Since arbitrary files aren't always Red Hat content, I think you should keep only the `following content types` version. Also, containers are certainly part of Pulp 3 and I wouldn't call that a Red Hat content type.

Reply:
I agree with Ewoud, I would keep it generic.

Reply:
Applied.